### DISCUSSION TITLE
Visual Studio 2019 showing yellow exclamation for .NET 5 Preview 8

### DISCUSSION THREAD

**Discussion by AspNetWebDev** [dotnet/aspnetcore#25806] [General] [answered]
Hello,

I recently installed .NET 5 Preview 8 and created a new ASP.NET Core MVC project in Visual Studio 2019 16.8 Preview 2.

For all the NuGet packages it is showing yellow exclamation mark. There are no compilation errors, no warnings. The application works fine. No errors. But VS keeps showing yellow exclamation. This problem comes only for .NET 5 projects. ASP.NET Core 3.1 projects don't give this problem.

![image](https://user-images.githubusercontent.com/16849146/92863358-73a87100-f419-11ea-844d-cd6e53193f35.png)

My project file is shown below :

```xml
<Project Sdk="Microsoft.NET.Sdk.Web">
  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="5.0.0-preview.8.20414.8" />
    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.0-preview.8.20407.4" />
  </ItemGroup>
</Project>
```

What's the reason and how to fix it ??

Thank you.

**Comment by Pilchie** [comment]
I believe this is a known issue.  @dsplaisted ?

**Comment by dsplaisted** [comment, marked as answer]
Yes, this is a preview 8 known issue: https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0-known-issues.md#preview-8-1
